lib/cn/cn_tree.c

```diff
@@ -671,11 +674,11 @@ cn_tree_query_common(
 }
 
 merr_t
-cn_tree_query(
+cn_tree_serialize(
     struct cn_tree *tree,
     const bool human,
     const bool kvsets,
-    cJSON *root)
+    cJSON *const root)
```

Comment:
This function name is misleading.  It doesn't actually serialize a cn tree.

Reply:
What is it serializing then?

Reply:
What @beckerg said

Reply:
Ok, I will use the term snapshot instead of serialize

Reply:
How about `_...serialize_rest_response` ?  or remove `cn_tree_` from the function name since it's `static`.

Oh, but it's not static.   Seems like it could be...

Reply:
It can't be static because a function in cn.c has to call it.

Reply:
cn_tree_stats_to_jsson()?  Can't think of anything better...

Reply:
I will just use `to_json